pkg/network/node/networkpolicy.go

```diff
@@ -42,6 +43,7 @@ type npNamespace struct {
 	name  string
 	vnid  uint32
 	inUse bool
+	flows []string
```

Comment:
You probably want to use the [`sets.String`](https://github.com/kubernetes/apimachinery/blob/master/pkg/util/sets/string.go) type from kubernetes here and in `syncNamespace`. It already has a `Difference` function.

Reply:
will give it a try, thx